packages/file/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+#### dev
```

Comment:
Do you not need to publish a new version right away?

Reply:
No. I need workarounds or something for https://github.com/dart-lang/sdk/issues/33459 in addition to this.

Reply:
Which I intend to write. Tonight or tomorrow.